src/Microsoft.Identity.Client/Instance/B2CAuthority.cs

```diff
@@ -37,7 +37,7 @@ internal class B2CAuthority : AadAuthority
         public const string Prefix = "tfp"; // The http path of B2C authority looks like "/tfp/<your_tenant_name>/..."
         public const string B2CCanonicalAuthorityTemplate = "https://{0}/{1}/{2}/{3}/";
         public const string OpenIdConfigurationEndpoint = "v2.0/.well-known/openid-configuration";
-        public const string B2CTrustedHost = "b2clogin.com";
+        private string[] B2CTrustedHosts = { "b2clogin.com", "b2clogin.cn" };
```

Comment:
what about BlackForest and US gov? Isn't B2C yet deployed there?

Reply:
Will follow up with B2C to get clarification on those.